src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala

```diff
@@ -78,6 +78,14 @@ class BitbucketClient(key: String, secretKey: String, token: String, secretToken
     value
   }
 
+  def postForm[T](request: Request[T], values: Map[String, Seq[String]])(implicit reader: Reads[T]): RequestResponse[T] = {
```

Comment:
We do not want anybody to use the post. The api is limited to the post json or post form data.